### PR TITLE
configure message severity from cli

### LINKED ISSUE #4: Allow to override message severity

### PR DESCRIPTION
**Pull request by bhirsz**
Closes #4

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -118,3 +118,18 @@ Format output message
 Output message can be defined with -f / --format argument. Default value::
 
     {source}:{line}:{col} [{severity}] {msg_id} {desc}
+
+Configure message severity
+--------------------------
+
+Configure message severity with -c/--configure argumen and message id or name::
```
Comment (mnojek) [inline README.rst:125]: Typo: `argumen`
I will also rephrase it to:
``Configure message severity with `-c/--configure` argument by providing message id or name::``

Comment (bhirsz) [reply README.rst:125]: Thanks! And those readme informations really need some better phrases ;)
